### PR TITLE
Titus as a runjob stage

### PR DESCRIPTION
**Pull request by tomaslin**
screen shot 2016-09-09 at 4 11 00 pm

and 

screen shot 2016-09-09 at 4 17 31 pm

### CODE REVIEW

File: app/scripts/modules/titus/pipeline/stages/runJob/titusRunJobStage.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.core.pipeline.stage.titus.runJobStage', [
+  require('./runJobExecutionDetails.controller.js')
+])
+  .config(function (pipelineConfigProvider) {
+    pipelineConfigProvider.registerStage({
+      provides: 'runJob',
+      useBaseProvider: true,
+      cloudProvider: 'aws',
```
Comment (tomaslin) [inline app/scripts/modules/titus/pipeline/stages/runJob/titusRunJobStage.js]: the cloud provider here is aws so as to not require titus to be needed as a cloud provider. it's a bit of a hack.

Comment (anotherchrisberry) [inline app/scripts/modules/titus/pipeline/stages/runJob/titusRunJobStage.js]: should this be `titus`?

Comment (anotherchrisberry) [inline app/scripts/modules/titus/pipeline/stages/runJob/titusRunJobStage.js]: ha

**Comment by anotherchrisberry** [comment]
LGTM

**Comment by duftler** [comment]
Isn't the Region control a bit wide in the screen-grab?

**Comment by anotherchrisberry** [comment]
@duftler I think it's beautiful.

**Comment by duftler** [comment]
image

**Pull request merged**